src/charts/stackedArea/StackedAreaComponent.test.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+
+import StackedAreaComponent from './StackedAreaComponent';
+import { stackedAreaData } from '../../helpers/testData';
+
+import stackedArea from './stackedAreaChart';
+
+describe('Stacked Area Chart', () => {
+
+    describe('render', () => {
+
+        it('should render the chart container', () => {
+            const wrapper = shallow(<StackedAreaComponent />);
+
+            let expected = 1;
+            let actual = wrapper.find('.stacked-area-container').length;
+
+            expect(actual).toEqual(expected);
```

Comment:
You sure you don't want to do:

```
expect(wrapper.find('.stacked-area-container')).toHaveLength(1)
```

?

Reply:
Nope, I prefer to use the Test Anything Protocol, keeping the assertion simple and not having to learn any new matcher.